docs/docs/installation/rasa-pro/rasa-pro-artifacts.mdx

```diff
@@ -12,15 +12,15 @@ import RasaProBanner from "@theme/RasaProBanner";
 
 <RasaProBanner />
 
-- **Rasa Plus**, a drop-in replacement for Rasa Open Source enterprise
-  deployments. Rasa Plus should be used to deploy your assistant
+- **Rasa Pro**, a drop-in replacement for Rasa Open Source enterprise
```

Comment:
This refers to Rasa Plus, although I think the whole sentence is a mixture of multiple concepts and quite confusing.

Reply:
@rgstephens ,
It looks great. I have a couple minor items and one area where the description is not clear, but we might have to live with that.

Reply:
> This refers to Rasa Plus, although I think the whole sentence is a mixture of multiple concepts and quite confusing.

Good change, updated